### PR TITLE
feat: add text codec

### PR DESCRIPTION
**Pull request by nic-chen**

### CODE REVIEW

File: codec/text.go [hunk 1]
```diff
@@ -0,0 +1,28 @@
+package codec
+
+import (
+        "github.com/shiningrush/droplet/data"
+        "net/http"
+)
+
+type Text struct {
```
Comment (ShiningRush) [inline codec/text.go]: It would be better to called `Empty`, because this codec should do nothing.

File: codec/text.go [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+                return err
+        }
+
+        ptr = bs
+
+        return nil
+}
+
+func (j *Text) Marshal(ptr interface{}) ([]byte, error) {
+        panic("MultipartForm not support marshal")
```
Comment (ShiningRush) [inline codec/text.go]: Fixed name match codec name.

File: codec/text.go (hunk 1 above)
Comment (nic-chen) [reply codec/text.go]: fixed

File: codec/text.go (hunk 2 above)
Comment (nic-chen) [reply codec/text.go]: fixed

**Review by ShiningRush — APPROVED** [review]

**Pull request merged**